modules/Bio/EnsEMBL/Production/Pipeline/FileDump/README.txt

```diff
@@ -275,14 +302,49 @@ Example 'softmasked' header:
 The <seq_type> will be 'chromosome' for chromosomal regions, and will
 be 'primary_assembly' for non-chromosomal regions.
 
+========================================================================
+ [homology] | <species>-<assembly>-<geneset>-<variation-source>.tsv.gz
+========================================================================
+The homologues of each query gene in a tab-separated format (tsv)
 
 ========================================================================
- [rnaseq] | *.bam
- [rnaseq] | *.bam.bai
  [rnaseq] | *.bam.bw
 ========================================================================
-BAM and bigWig files of RNA-seq data aligned against a genome. A README
+BigWig files of RNA-seq data aligned against a genome. A README
 file in the [rnaseq] directory contains details of the samples used in
-the alignments. This directory is absent if there are no RNA-seq
-alignments.
+the alignments. This directory is absent if there are no RNA-seq alignments.
+
+========================================================================
+ statistics | *_busco_short_summary.txt
+========================================================================
+BUSCO provides a quantitative assessment of the completeness of the gene set.
+The file contains a plain text summary of the lineage dataset used to score
+the gene set and the completeness results from BUSCO, run in protein mode,
+in BUSCO notation.
 
+========================================================================
+ variation | indexed_vep_cache | *.tar.gz
+ variation | vcf | <species>-<assembly>-<geneset>-<variation-source>.vcf.gz
+ variation | vcf | <species>-<assembly>-<geneset>-<variation-source>.vcf.gz.tbi
+ variation | vep | *.tar.gz
+========================================================================
+---
+vcf
+---
+This directory contains sorted and bgzip-compressed VCF (Variant Call Format) files:
+
+The data contained in these files is presented in VCF format. For more details about
+the format refer to:
+http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41
+
+    <species>-<assembly>-<geneset>-<variation-source>.vcf.gz
+        All variations from the current Ensembl Rapid release for this species
+    <species>-<assembly>-<geneset>-<variation-source>.vcf.gz.tbi
+        All variations tabix from the current Ensembl Rapid release for this species
+
+-----------------
+vep
+indexed_vep_cache
+-----------------
+
+This directory contains VEP tool utility files, for more information please refer to http://www.ensembl.org/info/docs/tools/vep/index.html
```

Comment:
These directories contain Ensembl VEP caches of the remapped Ensembl/GENCODE gene set to facilitate analysing your own variant data with Ensembl VEP.

Reply:
@jamie-m-a  Just updated the text, but kept the Reference to documentation, is it ok?